CoreFoundation/URL.subproj/CFURLSessionInterface.c

```diff
@@ -111,6 +111,14 @@ CFURLSessionEasyCode CFURLSession_easy_setopt_tc(CFURLSessionEasyHandle _Nonnull
     return MakeEasyCode(curl_easy_setopt(curl,  option.value, a));
 }
 
+CFURLSessionEasyCode CFURLSession_easy_setopt_blob(CFURLSessionEasyHandle _Nonnull curl, CFURLSessionOption option, void* data, size_t len) {
+    struct curl_blob stblob;
```

Comment:
This fails to build in CI because `struct curl_blob` (and `CURLOPT_SSLKEY_BLOB` and friends) are only available in libcurl >= 7.71.0. Let's add a macro to guard this, and do nothing/return `CFURLSessionEasyCodeOK` otherwise. (PR #4877 has a good example with the `NS_CURL_MISSING_CURLINFO_CAINFO` macro)